core/src/main/resources/swagger/rawls.yaml

```diff
@@ -3759,6 +3759,9 @@ definitions:
       expression:
         type: string
         description: Expression that resolves to one or more entities matching the entity type in the method configuration.
+      readFromCache:
+        type: boolean
+        description: Whether or not to read from cache for this submission
```

Comment:
Nitpick: I'd say "read from *call* cache" here, to be more specific. (This may be a running theme in the review...)